### PR TITLE
pyrokinetic spark

### PR DESCRIPTION
**Pull request by AysheDaArtist**
## About The Pull Request
<details>
<summary>
	Adds a new ability for all psions: Pyrokinetic Spark - Expend a single point of your psi essence to create a tiny flickering fire in your hand that will shine light and ignite combustible materials, can be thrown but will extinguish quickly

</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:

/:cl:

### CODE REVIEW

**Comment by Guidesu** [comment]
OMG YES!

File: code/modules/psionics/psionic_items/psi_temp_items.dm [hunk 1]
```diff
@@ -34,6 +34,52 @@
...
+	..()
+	holder = Maker
+	set_light(3)
+	START_PROCESSING(SSobj, src)
+
+/obj/item/flame/pyrokinetic_spark/Process()
+	..()
+	if(loc != holder) // We're no longer in the psionic's hand.
+		sleep(4)
+		visible_message("The [src.name] flickers away into nothingness.")
```
Comment (Guidesu) [inline code/modules/psionics/psionic_items/psi_temp_items.dm:79]:
```suggestion
		visible_message("The [src.name] flickers away as the fire fades into nothingness")
```
MY SUGGESTION

File: code/modules/psionics/psionic_items/psi_temp_items.dm [hunk 2]
```diff
@@ -34,6 +34,52 @@
...
+	burntime--
+	if(burntime < 1)
+		burn_out()
+		return
+	if(location)
+		location.hotspot_expose(700, 5)
+		return
+
+/obj/item/flame/pyrokinetic_spark/proc/burn_out()
+		visible_message("The [src.name] flickers away from your hand into nothingness.")
```
Comment (Guidesu) [inline code/modules/psionics/psionic_items/psi_temp_items.dm:64]:
```suggestion
		visible_message("The [src.name] flickers away from your hand like it was nothing.")
```

Comment (Guidesu) [reply code/modules/psionics/psionic_items/psi_temp_items.dm:64]: in general this messages are the same as for the other items so a bit of variety of flavour would be fine

**Comment by AysheDaArtist** [comment]
Yee, this was designed to be just the flame part of the omnitool, thus it costs less, but it still can't function as a welder.

It was designed to have scrolls be burned with this, if you want to have it heat up alchemy too, that's fine.

**Pull request merged**